src/components/elements/ingredient/FormEditMenuItemPreload.tsx

```diff
@@ -0,0 +1,300 @@
+                  }}>
+                    Back
+                </Button>
+              </FormControl>
+            </Grid>
+            <Grid item xs={12} sm={8}>
+              <div>
+
+
+                <FormControl component="fieldset" className={classes.formControl}>
+                  <FormLabel component="legend"></FormLabel>
+                  <Grid container spacing={3}>
+                    <Grid item xs={12} sm={4}>
+                        Select Ingredients
+                    </Grid>
+                    <Grid item xs={12} sm={4}>
+                        Select Measurment
+                    </Grid>
+                    <Grid item xs={12} sm={4}>
+                        Select Amount
+                    </Grid>
+                  </Grid>
+                  {IngredientListToPage.map((item:MenuIngredientForForm, index:number) =>
+                    <FormGroup key = {index}>
+                      <Grid container spacing={3}>
+
+                        <Grid item xs={12} sm={4}>
+                          <FormControlLabel
+                            control={<
+                              Checkbox
+                              defaultChecked={item.used}
+                              checked={item.used}
+                              value = {item.name}
+                              /*                              checked={item.used}*/
+                              onChange={() => {
+                                handleCheck(item.name);
+                              }}
+
+                              name="gilad" />}
+                            label={item.name}
+                          />
+                        </Grid>
+                        <Grid item xs={12} sm={4}>
+                          <Select
+                            defaultValue={item.measurement}
+                            // @ts-ignore
+                            value = {item.measurement}
+                            onChange={(option) => handleSelect(option, item.name)}
+                            options={Measurements}
+                          />
+                        </Grid>
+                        <Grid item xs={12} sm={4}>
+                          <TextField
+                            defaultValue={item.amount}
+                            value = {item.amount}
+                            type = 'number'
+                            id="Unit"
+                            label="Amount"
+                            onChange={handleAmount(item.name)}
+                            variant="filled"
+                          />
+                        </Grid>
+
+                      </Grid>
+                    </FormGroup>,
+                  )}
+
+                </FormControl>
+                <Button
+                  variant='contained'
+                  color='primary'
+                  style={styles.button}
+                  onClick={function() {
+                    if (true) {
```

Comment:
What is this doing? This will always work since you're saying "if(true)". Change this up.

Reply:
Good catch I needed to put a function in there